detection_rules/attack.py

```diff
@@ -44,6 +53,11 @@ def load_attack_gz():
         if item.get('revoked'):
             revoked[technique_id] = item
 
+        if item.get('x_mitre_deprecated'):
+            deprecated[technique_id] = item
+
+revoked = dict(sorted(revoked.items(), key=lambda x: x[0]))
+deprecated = dict(sorted(deprecated.items(), key=lambda x: x[0]))
```

Comment:
what's the point of sorting the dict items then putting it in a dict anyway?
also you don't need the key function, it's redundant

Reply:
since py3 keeps it sorted even as a dict. I will drop the `key`, nice catch, thanks 👍